lessons/2-methods.md

```diff
@@ -128,6 +129,143 @@ puts add_two(3)
 
 </details>
 
-## Homework
+### Homework
+
+[CodeWars - Can we divide it?](https://www.codewars.com/kata/5a2b703dc5e2845c0900005a/train/ruby)
+
+# German
+
+## Lesson 2 - Methods
+
+  - [DataTypen Fortsetzung](#datatypen-fortsetzung)
+    - [Boolean](#boolean)
+  - [Methoden](#methoden)
+    - [Allgemein](#allgemein)
+    - [Rückgabewerte](#rückgabewertreturn-value)
+    - [Mehr Spass mit Methoden](#mehr-spass-mit-methoden)
+  - [Hausaufgaben(Optional)](#hausaufgabenoptional)
+
+
+### DataTypen Fortsetzung
+
+Der erste Teil zu Datentypen befindet sich hier:  [DataTypen](/lessons/1-variables.md#datentypen)
```

Comment:
I'd suggest to be consistent here and use `Datentypen` rather than `DataTypen`:

```suggestion
### Datentypen Fortsetzung

Der erste Teil zu Datentypen befindet sich hier:  [Datentypen](/lessons/1-variables.md#datentypen)
```